cves/CVE-2011-1183.yml

```diff
@@ -43,7 +43,13 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The basic configurations for a Tomcat web application are set in the web.xml file. 
+  This file includes a config called "login-config" which sets a level of security 
+  for the application by whitelisting url's based on a user's role. This bug occurred 
+  when this configuration was not set and a configuration called "metadata-complete" 
+  was set to true, unintentionally allowing users to "bypass intended access restrictions" 
+  through HTTP requests.
```

Comment:
What access restrictions could be bypassed by exploiting this?

Reply:
It was difficult to find information on the specifics of this (probably for security reasons). I will look into finding more details if there are any out there.